tests/calc/test_basic.py

```diff
@@ -809,3 +810,32 @@ def test_altimeter_to_sea_level_pressure_hpa(array_type):
         [1009.963556, 1013.119712, 1015.885392, 1016.245615], 'hectopascal', mask=mask
     )
     assert_array_almost_equal(res, truth, 3)
+
+
+def test_xarray_zoom():
+    """Test xarray_zoom on 2D DataArray."""
```

Comment:
```suggestion
def test_zoom_xarray():
    """Test zoom_xarray on 2D DataArray."""
```
Small detail, but will help if you're in there already!

Reply:
You know, it would help if I actually stopped and looked at all of the files that were changed by this PR to catch those ahead of time, but alas. All corrected now!